### PR TITLE
Fix peer dependencies, fixes #11

### LINKED ISSUE #11: Missing react peer dependency
Doesn't compile for me with `Unable to resolve module `React` from `/Users/tolo/Repos/Cinuru/app/node_modules/@react-native-community/status-bar/js/StatusBar.js`: Module `React` does not exist in the Haste module map or in these directories:
  .../app/node_modules`

probably the missing React peer dependency declaration

### PR DESCRIPTION
**Pull request by MrLoh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
   "author": "Dmitriy Kovalenko <dmtr.kovalenko@outlook.com>",
   "license": "MIT",
   "peerDependencies": {
-    "react-native": "^0.41.2"
+    "react": "*",
+    "react-native": ">=0.41.2"
   },
```
Comment (dmtrKovalenko) [inline package.json:20]: Why >=?

**Review by cpojer — APPROVED** [review]
Thank you for the fixes!

**Comment by cpojer** [comment]
@dmtrKovalenko could you make a new release?

**Comment by dmtrKovalenko** [comment]
@cpojer I can, but I am not really sure that peer dependency react * is correct

**Comment by dmtrKovalenko** [comment]
And for me now everything seems to be working. Maybe because of yarn ?

File: package.json (hunk 1 above)
Comment (cpojer) [reply package.json:20]: Just looking at https://github.com/react-native-community/react-native-webview/blob/master/package.json#L23-L26 it definitely makes sense this way :)

**Comment by cpojer** [comment]
I'm unsure why it was working for you, but this PR is the right fix.

**Comment by dmtrKovalenko** [comment]
Anyway react: * is bad pattern.

**Comment by dmtrKovalenko** [comment]
If react will introduce any breaking change in example 17.0 - "Huston, we have a problem"

**Comment by cpojer** [comment]
I'm fine with changing it to ^16.0.

**Pull request merged**